new/mode-agnostic-data-collection.md

```diff
@@ -0,0 +1,549 @@
+---
+RFC: unassigned
+Author: Adam Leff <adamleff@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Mode-Agnostic Run Data Collection
+
+In order to provide increased clarity into a user's fleet operations, and to support Chef deployments that do not necessarily include the use of a Chef Server, this change aims to provide an easy and consistent way for the Chef Client to report run statistics to a data collection system. The initial statistics we look to collect include node state, converge start/end times, and converge details.
+
+## Motivation
+
+    As a Chef user who uses both Chef Server and Chef Client local mode,
+    I want to be able to collect data about my entire fleet regardless of their client operation type,
+    so that I may better understand the impacts of my changes and may better detect failures.
+
+## Specification
+
+Similar to how data is collected and reported for Chef Reporting, we expect to implement a new EventDispatch class/instance that collects data about the Chef Client run and reports it accordingly. Unlike Chef Reporting, the server that receives this data is **not** running on the Chef Server, allowing users to utilize this function whether they use Chef Server or not.
```

Comment:
It would be good to spell out here that since we're on solo we're not using chef-server authentication, so that requires a fundamental change.  Its sort of implicit and should be as easy as "2+2=4" to figure out, but my brain went running down other avenues and entirely missed it.